lib/honeybadger/config.rb

```diff
@@ -262,6 +262,27 @@ def load_plugin?(name)
       includes_token?(self[:plugins], name)
     end
 
+    def insights_enabled?
+      return false unless !(defined?(::Rails.application) && ::Rails.const_defined?("Console"))
+      !!self[:'insights.enabled']
+    end
+
+    def cluster_collection?(name)
+      return false unless insights_enabled?
+      return true if self[:"#{name}.insights.cluster_collection"].nil?
+      !!self[:"#{name}.insights.cluster_collection"]
+    end
+
+    def collection_interval(name)
+      return nil unless insights_enabled?
+      self[:"#{name}.insights.collection_interval"]
+    end
+
+    def load_plugin_insights?(name)
+      return false unless insights_enabled?
+      self[:"#{name}.insights.enabled"] != false
```

Comment:
Why does this check not use the `!!` approach used elsewhere (such as on line 267)?

Reply:
This is because `#{name}.insights.enabled` can be nil (as in not configured), which then the methods needs to return true.

I'm going to add a guard clause to make this clear (took me a minute to remember why I had did this).